### PR TITLE
docs(storage-reverse-image-search): Fix a typo

### PR DESCRIPTION
**Pull request by nohe427**
s/sit/set

### CODE REVIEW

**Review by pr-Mais — APPROVED** [review]
Thank you!

**Comment by pr-Mais** [comment]
CI is failing for the README since PREINSTALL.md and README aren't in sync now.

To fix this, please apply the same change in PREINSTALL, and run:
```
npm run generate-readme 
```
in `storage-reverse-image-search/functions`.

File: storage-reverse-image-search/README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ This extension uses other Firebase and Google Cloud Platform services, which hav
 
 * Approximate number of neighbors: The approximate number of neighbors to return in the response. [Read more about this parameter here](https://cloud.google.com/vertex-ai/docs/matching-engine/configuring-indexes#nearest-neighbor-search-config).
 
-* Batch size: The batch size used to generate feature vectors for existing images. The larger the batch size, the more time and memory are required. Do not sit a size larger than 64.
+* Batch size: The batch size used to generate feature vectors for existing images. The larger the batch size, the more time and memory are required. Do not set a size larger than 64.
```
Comment (pr-Mais) [inline storage-reverse-image-search/README.md:91]: Apply this fix to [PREINSTALL.md](https://github.com/GoogleCloudPlatform/firebase-extensions/blob/next/storage-reverse-image-search/PREINSTALL.md)

**Review by pr-Mais — CHANGES_REQUESTED** [review]

File: storage-reverse-image-search/README.md (hunk 1 above)
Comment (nohe427) [reply storage-reverse-image-search/README.md:92]: I could not find the `npm run generate-readme` command for some reason

Comment (nohe427) [reply storage-reverse-image-search/README.md:92]: I did push the update to s/sit/set

Comment (pr-Mais) [reply storage-reverse-image-search/README.md:92]: LGTM thanks!

You can find it in [functions/package.json](https://github.com/GoogleCloudPlatform/firebase-extensions/blob/ea2ed5c2d4246be309a33e26428d16fdd6de18c7/palm-secure-backend/functions/package.json#LL9C6-L9C21)

**Comment by cabljac** [comment]
closing in favour of https://github.com/GoogleCloudPlatform/firebase-extensions/pull/109

seems like there might be a CI issue here, looking into it now

**Pull request closed without merging**